.github/workflows/benchmark.yml

```
@@ -0,0 +1,48 @@
+# Note: this workflow is currently being tested,
+# and so is only activated for the benchmark-test-cjs branch
+
+name: Performance benchmarks
+
+on:
+  push:
+    branches:
+      - benchmark-test-cjs
+
+jobs:
+
+  benchmarks:
+    runs-on: ${{ matrix.os }}
+
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-latest]
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python
+      uses: actions/setup-python@v1
```

```suggestion
      uses: actions/setup-python@v2
```
We switched all other workflows to v2.